### ISSUE TITLE
NVDA cannot see  pop up information on product in virtual mailer at New World

### ISSUE THREAD

**Issue by NZGene** [nvaccess/nvda#11330]
### Steps to reproduce: 
Open google chrome browser
Type in newworld.co.nz
Press Enter
From top of page, Tab (approximately 5 times) to virtual mailer
Press Enter
Virtual mailer should pop up (this will be in the pdf viewer in those browsers)
From top of page, press B for button until you hear "next page"
Arrow down about 5 or 6 times until you hear a product (like cheese)
Press the Enter key on that product
A pop up will appear that NVDA cannot see
It will have a picture of the product, a brief description and the product's price
Underneath  it has Find Store and Add to list which NVDA is unable to read
If pressing the Enter key on the product in the mailer with Jaws or Window Eyes, this page is read out by the other screen readers when you jump to it with the letter H
You can not do this with NVDA or even see that information on the screen

Please note: Being a weekly virtual mailer, I cannot link straight to the pdf viewer page as I think it would change weekly!

### Actual behavior:
NVDA does not read the pop up information about the product in the virtual mailer

### Expected behavior:
I would expect NVDA to be able to get to this information and read it (like Jaws and Window Eyes can)

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
2020.1

#### Windows version:
Version 1903 OS Build 18362.592

#### Name and version of other software in use when reproducing the issue:
Google chrome 83.0.4103.116 official build 64 bit 
Microsoft Edge 83.0.478.58 official build 64 bit

#### Other information about your system:
System Name	HP_NOTEBOOK
System Manufacturer	Hewlett-Packard
System Model	HP Pavilion g6 Notebook PC
System Type	x64-based PC
Processor	AMD A6-4400M APU with Radeon(tm) HD Graphics, 2700 Mhz, 2 Core(s), 2 Logical Processor(s)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. Exactly the same behaviour as above (not seeing the pop up with the product information, price etcetera).

#### If addons are disabled, is your problem still occuring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes, but it didn't change anything
[nvda.log](https://github.com/nvaccess/nvda/files/4869122/nvda.log)

**Comment by seanbudd** [comment]
Is this still an issue? Have you reported this to `newworld.co.nz`?

**Comment by seanbudd** [comment]
Closing as stale / needs a fix from the website authors
